src/main/java/io/fabric8/vertx/maven/plugin/mojos/RunMojo.java

```diff
@@ -33,8 +36,111 @@
     requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
 public class RunMojo extends AbstractRunMojo {
 
+    /**
```

Comment:
I thought the other way as much simpler, then the user will know what they are doing e.g. 
```
   <ha>true</ha>
   <haGroup>demo</haGroup>
  <quorum>2</quorum>
```